### PR TITLE
[TACHYON-1001] remove unused variable LOG

### PR DESCRIPTION
**Pull request by xiaocai00**
https://tachyon.atlassian.net/browse/TACHYON-1001

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

File: common/src/main/java/tachyon/util/CommonUtils.java [hunk 1]
```diff
@@ -21,13 +21,11 @@
 import java.util.List;
 
 import org.slf4j.Logger;
```
Comment (gpang) [inline common/src/main/java/tachyon/util/CommonUtils.java]: Is this import still required?

**Comment by gpang** [comment]
Thanks for the PR! I just had a minor question.

**Comment by calvinjia** [comment]
Jenkins, add to whitelist.

File: common/src/main/java/tachyon/util/CommonUtils.java (hunk 1 above)
Comment (xiaocai00) [inline common/src/main/java/tachyon/util/CommonUtils.java]: @gpang Logger is used by some other functions, e.g. sleepMs

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**